app/src/store/store.ts

```diff
@@ -49,8 +49,8 @@ export class Store {
   settingsStore = new SettingsStore(this);
   sessionStore = new SessionStore(this);
 
-  /** the store which synchronizes with the browser history */
-  router: RouterStore;
+  // /** the store which synchronizes with the browser history */
+  // router: RouterStore;
```

Comment:
Unsure whether we need `RouteStore` as `useLocation` hook exposes `location` to all components within the `*Router`. Perhaps I can inject `location` when we invoke `createStore` in `App`.

Reply:
I'm fine with getting rid of this if we no longer needed.